includes/definitions/discovery/gigabyte-bmc.yaml

```diff
@@ -0,0 +1,40 @@
+mib: GIGABYTE-BMC
+modules:
+    os:
+        hardware: GIGABYTE-BMC::fruPartNumber.0
+        serial: SNMPv2-MIB::snmpSetSerialNo.0
+    processors:
+        data:
+            -
+                oid: processorCollection
+                num_oid: '.1.3.6.1.4.1.15370.552.2.1.1'
+
```

Comment:
This collection is not working as I would like it to. I guess I am not well enough versed to know what is supposed to happen here.
![image](https://user-images.githubusercontent.com/24249616/95211780-44550c00-07ed-11eb-8a2d-6ab9ece669f2.png)
This is what is being given by discovery.php in verbose mode. I get the following error/warning.
```
Warning: preg_match() expects parameter 2 to be string, array given in /opt/librenms/LibreNMS/Device/Processor.php on line 236
SQL[select * from `locations` where `locations`.`id` = ? limit 1 [2] 0.39ms] 
```
I would like to display the model numbers and iXXX of the CPU in the front-end.
But I am wondering if this OID does not pertain to the % utilisation rather than model type etc.?

Reply:
This data feels like it should be in the inventory section.